### PR TITLE
Issue 152

### LINKED ISSUE #152: Add hemonc.org CC BY data
https://dataverse.harvard.edu/dataverse/HemOnc

### PR DESCRIPTION
**Pull request by jsstevenson**
Close #152

### CODE REVIEW

File: tests/unit/test_merge.py [hunk 1]
```diff
@@ -48,6 +49,23 @@ def compare_merged_records(actual: Dict, fixture: Dict):
...
+            set(fixture['approval_status'])
+    assert ('approval_year' in actual) == ('approval_year' in fixture)
+    if 'approval_year' in actual or 'approval_year' in fixture:
+        assert set(actual['approval_year']) == set(fixture['approval_year'])
+    assert ('fda_indication' in actual) == ('fda_indication' in fixture)
+    if 'fda_indication' in actual or 'fda_indication' in fixture:
+        actual_inds = actual['fda_indication'].copy()
+        fixture_inds = fixture['fda_indication'].copy()
+        assert len(actual_inds) == len(fixture_inds)
+        print(actual_inds)
```
Comment (korikuzma) [inline tests/unit/test_merge.py:64]:
```suggestion
```

File: README.md [hunk 2]
```diff
@@ -65,38 +65,29 @@ export RXNORM_API_KEY={rxnorm_api_key}
...
-To create merged concept groups, use the `--update_merged` flag with the `--update_all` flag.
+The `/normalize` endpoint relies on merged concept groups.  The `--update_merged` flag generates these groups:
 
 ```commandline
-python3 -m therapy.cli --update_all --update_merged
+python3 -m therapy.cli --update_merged
 ```
 
 #### Specifying the database URL endpoint
+
```
Comment (korikuzma) [inline README.md:90]: I think it would be useful to add instructions for the manual data download.

Comment (korikuzma) [reply README.md:90]: As well as what data for disease needs to be loaded

File: therapy/etl/hemonc.py [hunk 3]
```diff
@@ -0,0 +1,233 @@
...
+                concept_id = f'{NamespacePrefix.HEMONC.value}:{row[3]}'
+                therapies[row[3]] = {
+                    'concept_id': concept_id,
+                    'label': row[0],
+                    'trade_names': [],
+                    'aliases': [],
+                    'other_identifiers': [],
+                }
+            elif row_type == 'Brand Name':
+                brand_names[row[3]] = row[0]
```
Comment (korikuzma) [inline therapy/etl/hemonc.py:111]: I haven't checked this, but you probably have. Is it possible that a concept could have multiple unique brand names?

File: therapy/query.py [hunk 4]
```diff
@@ -354,10 +362,14 @@ def _record_order(self, record: Dict) -> (int, str):  # TODO refactor?
...
-        elif src == SourceName.CHEMIDPLUS.value:
+        elif src == SourceName.HEMONC.value:
             source_rank = 3
-        else:
+        elif src == SourceName.DRUGBANK.value:
             source_rank = 4
+        elif src == SourceName.CHEMIDPLUS.value:
+            source_rank = 5
+        else:
+            source_rank = 6
```
Comment (korikuzma) [inline therapy/query.py:372]: Could this be changed to `source_rank = SourcePriority[src].value` ?

Comment (korikuzma) [inline therapy/query.py:372]: Could this be changed to `source_rank = SourcePriority[src].value` ?

File: therapy/schemas.py [hunk 5]
```diff
@@ -47,10 +47,20 @@ class PhaseEnum(IntEnum):
     approved = 4
 
 
+class HasIndication(BaseModel):
```
Comment (korikuzma) [inline therapy/schemas.py:50]: Should a schemas example be added?

**Review by korikuzma — APPROVED** [review]
Minor changes that I think would be useful (if you disagree, you can disregard), but otherwise great job!

File: therapy/etl/hemonc.py (hunk 3 above)
Comment (jsstevenson) [reply therapy/etl/hemonc.py:111]: yep! here, `row[3]` is the ID for the brand name (string in `row[0]`). The relations CSV then ties this ID back to the ID for the concept.

**Pull request merged**